x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect } from 'react';
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiTitle,
+  EuiPageContentBody,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiCopy,
+  EuiButtonIcon,
+  EuiSpacer,
+  EuiButton,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import { SetAppSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import {
+  CredentialsLogic,
+  ICredentialsLogicActions,
+  ICredentialsLogicValues,
+} from './credentials_logic';
+import { externalUrl } from '../../../shared/enterprise_search_url/external_url';
+import { CredentialsList } from './credentials_list';
+import { CredentialsFlyout } from './credentials_flyout';
+
+export const Credentials: React.FC = () => {
+  const { initializeCredentialsData, resetCredentials, toggleCredentialsForm } = useActions(
+    CredentialsLogic
+  ) as ICredentialsLogicActions;
+
+  const { dataLoading, showCredentialsForm } = useValues(
+    CredentialsLogic
+  ) as ICredentialsLogicValues;
+
+  useEffect(() => {
+    initializeCredentialsData();
+    return () => {
+      resetCredentials();
+    };
+  }, []);
+
+  // TODO
+  // if (dataLoading) { return <Loading /> }
+  if (dataLoading) {
+    return null;
+  }
+  return (
+    <>
+      <SetPageChrome isRoot />
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>
+              <FormattedMessage
+                id="xpack.enterpriseSearch.appSearch.credentials.title"
+                defaultMessage="Credentials"
+              />
+            </h1>
+          </EuiTitle>
+        </EuiPageHeaderSection>
+      </EuiPageHeader>
+      <EuiPageContentBody>
+        {showCredentialsForm && <CredentialsFlyout />}
```

Comment:
This actually isn't working in it's current state... it's being overshadowed by it's own shadow, and I'm not sure why:

![screencapture-localhost-5601-app-enterprise-search-app-search-credentials-2020-10-02-16_16_08](https://user-images.githubusercontent.com/1427475/94966067-a0a4ec80-04ca-11eb-864c-9d28200225f7.png)